### PR TITLE
BCI-1259: [REVISED] upgrade test

### PR DESCRIPTION
**Pull request by augustbleeds**
Based off of https://github.com/smartcontractkit/chainlink-starknet/pull/256 but git history got messy

### CODE REVIEW

File: contracts/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "compile:cairo": "scarb --profile release build",
```
Comment (archseer) [inline contracts/package.json]: https://github.com/smartcontractkit/chainlink-starknet/pull/240#discussion_r1189370695

Comment (cfal) [reply contracts/package.json]: +1, fyi the build contract actions in the Makefile were renamed to `build-cairo-contracts` and `build-sol-contracts` for clarity

**Comment by archseer** [comment]
What's still left here? Tests might pass after a rebase

**Comment by archseer** [comment]
Looks obsolete by now

**Pull request closed without merging**